app.js

```diff
@@ -1,21 +1,19 @@
 const express = require('express');
 const app = express();
-const { fetchCategories, fetchReviews } = require('./controllers/get-controllers');
-const { handleServerErrors } = require('./error-handlers');
+const { fetchCategories, fetchReviews, fetchReview } = require('./controllers/get-controllers');
+const { handlesServerErrors, handlesInvalidPath, handles404Errors, handles400Errors } = require('./error-handlers');
 
 // app.use(express.json());
 
 app.get('/api/categories', fetchCategories);
 
 app.get('/api/reviews', fetchReviews)
 
+app.get('/api/reviews/:review_id', fetchReview)
 
-// make this generic, not just for get method
-app.get('/*', function (req, res) {
```

Comment:
I would still keep your default /* path here - using app.all that gives a 404. But just keep your custom 404s in the error file

Reply:
just a bit confused becuase that's the way jim did it so I thought mine was wrong. Is one way better than another?